src/splitseq_combine.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Change to `#!/usr/bin/bash` or `#!/usr/bin/env bash`.

Reply:
Will do.